sbws/globals.py

```diff
@@ -53,6 +53,10 @@
 
 BW_LINE_SIZE = 510
 
+# This number might need adjusted depending on the percentage of circuits and
+# HTTP requests failures.
+MAXIMUM_NUMBER_DESTINATION_FAILURES = 10
```

Comment:
Will operators want to configure this value?

Reply:
Created #29589